src/index.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.scss';
-import App from './App';
+import { Mars } from './InteractiveMars/Mars';
 
 ReactDOM.render(
     <React.StrictMode>
-        <App />
+        <Mars />
```

Comment:
I think we will want the app - that has the router and stuff in it.
this should go inside the app instead.

(you might need to merge in master)